sf-stream/src/protos/extractor.proto

```diff
@@ -0,0 +1,275 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+message Transaction {
+  uint64 timestamp = 1;
+  uint64 version = 2;
+  TransactionInfo info = 3;
+  uint64 epoch = 4;
+  uint64 block_height = 5;
+
+  enum TransactionType {
+    GENESIS = 0;
+    BLOCK_METADATA = 1;
+    STATE_CHECKPOINT = 2;
+    USER = 3;
+  }
+  
+  TransactionType type = 6;
+
+  oneof txn_data {
```

Comment:
I think also here plain `data` or `payload` is better ultimately.

Also, something I forget to discuss with you guys, in the future, do you forecast adding more transaction type's? I imagine that if it's the case it should be fairly limited as much happen in the core system contracts which means more transactions type is probably not something that will happen at the core level.

Reply:
we don't expect to add or remove transaction types, but it can happen- certainly not with any frequency though